packages/fxa-auth-server/lib/routes/oauth.js

```diff
@@ -197,11 +197,15 @@ module.exports = (log, config, oauthdb, db, mailer, devices) => {
           const newSessionToken = await db.createSessionToken(
             sessionTokenOptions
           );
+          // the new session token information is later
+          // used later in 'newTokenNotification' to attach it to device records
```

Comment:
nit: "later used later" is awkward; I'd phrase this along the lines of:

```
// The session token info is used by `newTokenNotification` when creating a device record,
// so make sure it reflects the newly-created session token.
```

Assuming I correctly understand what's happening here, that is :-)